lib/stdlib/doc/src/re.xml

```diff
@@ -2478,6 +2477,11 @@ foo\Kbar</code>
       <tag>xdigit</tag><item>Hexadecimal digits</item>
     </taglist>
 
+    <p>There is another character class, <c>ascii</c>, that erroneously matches
+      Latin-1 characters instead of the 0-127 range specified by POSIX. This
+      cannot be fixed without altering the behaviour of other classes, so we
+      recommend matching the range with <c>[\x01-\x7f]</c> instead.</p>
```

Comment:
```suggestion
      recommend matching the range with <c>[\x00-\x7f]</c> instead.</p>
```

Reply:
`\x00` badargs when given in a pattern, and I thought zeroes were illegal in subjects, but apparently not:

```erlang
1> re:run(<<0,"ü"/utf8>>, "[\x01-\x7f]", []).
nomatch
```

I've changed the pattern to `[\\0-\x7F]` instead. Thanks for pointing it out :)